### PR TITLE
[UVHD Tenant Admin] Add admin cmdlet to delete and copy UVHD

### PR DESCRIPTION
**Pull request by arcadiahlyy**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@arcadiahlyy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hele). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ServiceManagement/RemoteApp/Commands.RemoteApp.Test/Collection/RemoteAppCollection.cs [hunk 1]
```diff
@@ -29,14 +30,14 @@ namespace Microsoft.WindowsAzure.Commands.RemoteApp.Test
     using System.Management.Automation;
     using System.Threading;
     using System.Threading.Tasks;
-    using Xunit;
 
     // Get-AzureRemoteAppCollectionUsageDetails, Get-AzureRemoteAppCollectionUsageSummary, 
+    [TestClass]
```
Comment (hovsepm) [inline src/ServiceManagement/RemoteApp/Commands.RemoteApp.Test/Collection/RemoteAppCollection.cs]: @arcadiahlyy do NOT convert Xunit tests to MS tests. Please revert this change back
